contracts/SafeProtocolMediator.sol

```diff
@@ -133,7 +152,7 @@ contract SafeProtocolMediator is ISafeProtocolMediator {
      * @param module ISafeProtocolModule A module that has to be enabled
      * @param allowRootAccess Bool indicating whether root access to be allowed.
      */
-    function enableModule(address module, bool allowRootAccess) external noZeroOrSentinelModule(module) {
+    function enableModule(address module, bool allowRootAccess) external noZeroOrSentinelModule(module) onlyPermittedModule(module) {
```

Comment:
we can possibly remove `noZeroOrSentinelModule(module)` is registry can guarantee that `address(0x1)` and `address(0x1)` are never permitted

Reply:
lets keep it for now, as for formal verification and auditing having this check makes it a proper invariant